arabic/journey to the the unknown

```diff
@@ -0,0 +1 @@
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.